tests/unit/amo/components/TestAddon.js

```diff
@@ -557,4 +557,26 @@ describe('mapStateToProps', () => {
 
     expect(installStatus).toEqual(UNKNOWN);
   });
+
+  it('sets random addon object properties', () => {
```

Comment:
Rather than "sets random addon props" (which makes it seem like maybe *literally* random props are set) I would say "passes all addon object properties".

A link to the issue explaining why would help too.